### ISSUE TITLE
blog post feedback

### ISSUE THREAD

**Issue by btopro** [haxtheweb/issues#2061]
- [x] inline widgets at times are lacking the content of what was highlighted for the wrapping
- [x] longdesc on a11y-gif-player should be loaded on configure tab
- [x] haxor-slevin blog appears to not be flipping to correct address; this is most likely a static vercel / gh-pages publishing issue but could be the interplay of this with our routing engine loading too quickly :shrug: (total guess)
- [ ] inline merlin is blank sometimes. it's inconsistent, may just be related to the infinite list lit labs thing we use... which does claim to be in labs so..... next release of lit has patches for virtualizer so maybe it'll fix it
- [x] instructional intention should come below colors
- [x] colors and longer design system fields should render in their own collapse within the category to save space
- [x] below nodejs errors to clean up (may just be local issue since it's a very old package w/ references that might be dated
- [x] errors are causing manifest / other templated files on major saves that rebuild these files to fail, leaving {{}} in output. Users won't notice and sites still work, but its breaking site manifest / other things that can impact routing position / base path

```
: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/16x16-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/192x192-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/192x192-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/192x192-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/16x16-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/16x16-odl-logo.png'
Error: ENOENT: no such file or directory, stat '/media/bto108a/AtticStorage/git/btopro/blog/files/haxcms-managed/192x192-odl-logo.png'
/home/bto108a/.npm/_npx/6f96bf50b660cee9/node_modules/@haxtheweb/haxcms-nodejs/dist/lib/HAXCMS.js:1158
          let image = new ImageResize(path + this.manifest.metadata.site.logo);
                      ^

ReferenceError: ImageResize is not defined
```
